Dockerfile

```diff
@@ -1,17 +1,50 @@
-# docker build -t firmadyne .
-FROM ubuntu:14.04
+# docker build -t firmadyne . && docker run -v $(pwd)/images:/firmadyne/images -it --privileged firmadyne
+FROM --platform=amd64 ubuntu:22.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV FIRMADYNE_INSTALL_DIR=/firmadyne
 
 # Update packages
-RUN apt-get update && apt-get upgrade -y && apt-get install -y sudo
+RUN apt-get update && apt-get upgrade -y && apt-get install -y sudo adduser
 
 # Create firmadyne user
 RUN useradd -m firmadyne
 RUN echo "firmadyne:firmadyne" | chpasswd && adduser firmadyne sudo
 
-# Run setup script
-ADD setup.sh /tmp/setup.sh
-RUN /tmp/setup.sh
-ADD startup.sh /firmadyne/startup.sh
+RUN apt-get install -y busybox-static fakeroot git \
```

Comment:
Why the change to not run the setup script?

Reply:
The main reason is docker layer caching -- if the setup script fails at any point, then docker cannot cache partial progress of the installation.

Reply:
I also have a personal preference that the `Dockerfile` and `setup.sh` script shouldn't be interdependent. It is unclear for example, that if I modify `setup.sh`, I cannot have any interactive behavior allowed: I would expect `setup.sh` to be run manually by a user, and the `Dockerfile` to be run automatically.

Reply:
Hmm ok, I don't really use Docker myself so I'm not too familiar with it, my main concern is that the setup steps here could diverge with those in the setup.sh script, is there any way to import or include some commands from a file shared with the setup.sh script?

Reply:
That would be possible, but not something I would be interested in doing unfortunately. 

Seeing as this project looks to be in maintenance mode, it makes sense to me to provide a more modern, working example for people to use instead of waiting for someone to do that.

When updates are made to setup.sh, ensuring that the developer updates the Dockerfile alongside it would be pretty easy.

Additionally, upstream projects like https://github.com/pr0v3rbs/FirmAE use a similar split of setup.sh/Dockerfile.